packages/react-menu/src/MenuRadioItems.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { Menu, MenuTrigger, MenuList, MenuItemRadio, MenuPopover } from './index';
+
+import { Button } from '@fluentui/react-button';
+import {
+  Cut20Regular as CutIcon,
+  ClipboardPaste20Regular as PasteIcon,
+  Edit20Regular as EditIcon,
+} from '@fluentui/react-icons';
+
+export const RadioItems = () => {
+  return (
+    <Menu>
+      <MenuTrigger>
+        <Button>Toggle menu</Button>
+      </MenuTrigger>
+      <MenuPopover>
+        <MenuList>
+          <MenuItemRadio icon={<CutIcon />} name="font" value="segoe">
```

Comment:
Might be worth explaining the `[name;value]` concept. Or do you expect people to understand that from HTML?

Reply:
I thought it would have been quite normal from `input` checkboxes and radios 😅 I can add an explainer and link to HTML standards

Reply:
Added a link to the official MDN docs and also added a sample code snippet of HTML inputs